src/nvim/bytes.c

```diff
@@ -0,0 +1,21 @@
+// Low-level functions with minimal dependencies, side-effects, etc.
+
+#include <assert.h>
+#include <inttypes.h>
+
+#include "nvim/vim.h"
+#include "nvim/bytes.h"
+
+#ifdef INCLUDE_GENERATED_DECLARATIONS
+# include "bytes.c.generated.h"
+#endif
+
+/// Writes time_t to "buf[8]".
+void time_to_bytes(time_t time_, uint8_t buf[8])
```

Comment:
Maybe, but this function don't really wrap any stdlib function. It might be reasonable to extract the more byte-manipulation functions in `memory.c` (from the basic allocation/freeing wrapper functions) to `bytes.c`, but then again, `memory.c` isn't really that big or bloated. But anyway that could be later PR, I just thought this function seemed very lonely for the moment :)